wazuh/wazuh-dashboard-plugins
# Develop logic of a new index for the fim module

### Description

Develop logic of a new index for the fim module
 
### Issues Resolved

#6210 

### Evidence

<img width="1792" alt="image" src="https://github.com/wazuh/wazuh-dashboard-plugins/assets/99441266/ef6d39c3-9a76-4cfb-88c2-c584aed6a489">

<img width="1792" alt="image" src="https://github.com/wazuh/wazuh-dashboard-plugins/assets/99441266/028654aa-8f8f-44d7-94c5-f355077260f8">

<img width="1792" alt="image" src="https://github.com/wazuh/wazuh-dashboard-plugins/assets/99441266/c5e1c0d0-571b-491d-9bb3-9f4cf30adf42">

### Test

- Verify that when you load the health check you get the index pattern warning of fim

- Verify that when you go to the dashboard tab it does not show visualisations as the index pattern is not available.

- Verify that the inventory tab does not load information.

### Check List
